libstorage/Common.h

```diff
@@ -87,6 +90,15 @@ const int CODE_TABLE_FIELDVALUE_LENGTH_OVERFLOW = -50006;
 const int CODE_TABLE_DUMPLICATE_FIELD = -50007;
 const int CODE_TABLE_INVALIDATE_FIELD = -50008;
 
+enum SQLFieldType : int8_t
+{
+    MediumBlobType = 0,
+    LongBlobType = 1,
+    MediumStringType = 2,
+    LongStringType = 3,
+};
+extern std::map<SQLFieldType, std::string> SQLFieldTypeName;
```

Comment:
net time please split the PR into 2 pr, one is modifying mysql type, one is adding getBlockHeader, so the reviewer can read faster

Reply:
Ok, got it.